bundles/org.openhab.ui/web/src/components/pagedesigner/sitemap/dslUtil.js

```diff
@@ -11,6 +11,13 @@ function writeWidget (widget, indent) {
         dsl += ' sendFrequency=' + widget.config[key]
       } else if (key === 'forceAsItem') {
         dsl += ' forceasitem=' + widget.config[key]
+      } else if (key === 'staticIcon') {
+      } else if (key === 'icon') {
```

Comment:
This will do nothing in case of `key === 'staticIcon'` (empty block); is it intended? What's the point of it if so?

Reply:
Thanks for noticing, must have thought I was using a Java case statement.